eng/SourceBuild.props

```diff
@@ -0,0 +1,25 @@
+<Project>
+
+  <PropertyGroup>
+    <GitHubRepositoryName>nuget-client</GitHubRepositoryName>
+    <SourceBuildManagedOnly>true</SourceBuildManagedOnly>
+
+    <VersionPrefix>1.0.0</VersionPrefix>
+  </PropertyGroup>
+
+  <Target Name="ApplySourceBuildPatchFiles"
```

Comment:
I see there are release/5.0 patches being used today.  Are these still needed?  This target is not necessary it the patches have already been integrated.

Reply:
Removed these, the patches have been integrated.